test/python/tensorflow/README.md

```diff
@@ -5,8 +5,9 @@ tf_unittest_runner is primarily used to run tensorflow python unit tests using n
 ## What can be tested
 
  - Python tests using Tensorflow with nGraph embedded
- - Python tests by patching Tensorflow to allow runnings tests using nGraph.
-	```tf_unittest_ngraph.patch```
+ - Python tests by patching Tensorflow to allow runnings tests using nGraph. 
+   Apply the patch ```tf_unittest_ngraph.patch``` in your installed tensoflow file:"<local virtualenv>/local/lib/<python>/site-packages/tensorflow/python/framework/test_util.py" 
+	
```

Comment:
In the documentation it says:
```
--tensorflow_path is a required argument and must be passed to specify the location of Tensorflow installation
```
Can `tensorflow_path` be installed TF or does it have to be source TF?

If the 2 are separate, we need to make that separate explicit in the instructions (installed TF for patch and source TF for `--tensorflow_path`). Can you please check if installed TF path might work for `--tensorflow_path`. I think I did, but it was not able to grab the tests